mozilla/fx-private-relay
# fix MPP-2305: make inbound_call respond 200 with <Say>

This PR fixes MPP-2305.

How to test:
1. Set your relay number to "blocking"
2. Call your relay number
   * [ ] You should hear "Sorry, that number is not available."
